lib/plugins/request_expiry.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var assert = require('assert-plus');
+var GatewayTimeoutError = require('./../errors').GatewayTimeoutError;
+
+/**
+ * A request expiry will use the headers to tell if the
+ * incoming request has expired or not.  The header is
+ * expected to be in absolute time since the epoch.
+ * @param {Object} options
```

Comment:
Can you add a description of what options is and also add @param {string} options.header with a description?